unix/src/parse.c

```diff
@@ -539,7 +585,13 @@ static void parseSendRequestInternal(
         }
     }
 
-    int urlSize = strlen("https://api.parse.com");
+    int urlSize = 0;
+    if(clientInternal->serverURL != NULL){
```

Comment:
If you refactor the initialize method, this change becomes trivial and you don't need the if